<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>emailPage</class>
 <widget class="QWidget" name="emailPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Электронная почта</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblSMTPServer">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>SMTP &amp;сервер</string>
     </property>
     <property name="buddy">
      <cstring>edtSMTPServer</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="edtSMTPServer">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblSMTPEncryption">
     <property name="text">
      <string>&amp;Шифрование</string>
     </property>
     <property name="buddy">
      <cstring>cmbSMTPEncryption</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cmbSMTPEncryption">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>нет</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TLS/STARTTLS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SSL</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblSMTPPort">
     <property name="text">
      <string>По&amp;рт</string>
     </property>
     <property name="buddy">
      <cstring>edtSMTPPort</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="edtSMTPPort">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>25</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblMailAddress">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Адрес эл. почты</string>
     </property>
     <property name="buddy">
      <cstring>edtMailAddress</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="edtMailAddress">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="chkSMTPAuthentification">
     <property name="text">
      <string>на сервере требуется аутентификация</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblSMTPLogin">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Логин</string>
     </property>
     <property name="buddy">
      <cstring>edtSMTPLogin</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="edtSMTPLogin">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>26</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblSMTPPassword">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Пароль</string>
     </property>
     <property name="buddy">
      <cstring>edtSMTPPassword</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="edtSMTPPassword">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <spacer name="horizontalSpacer_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>26</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0">
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>118</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtSMTPServer</tabstop>
  <tabstop>cmbSMTPEncryption</tabstop>
  <tabstop>edtSMTPPort</tabstop>
  <tabstop>edtMailAddress</tabstop>
  <tabstop>chkSMTPAuthentification</tabstop>
  <tabstop>edtSMTPLogin</tabstop>
  <tabstop>edtSMTPPassword</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chkSMTPAuthentification</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblSMTPLogin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSMTPAuthentification</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblSMTPPassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSMTPAuthentification</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtSMTPLogin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSMTPAuthentification</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtSMTPPassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
